scripts/setup_release_automation

```diff
@@ -28,6 +28,12 @@ CHANNEL_ENVIRONMENTS = {
                     "packageFormat": "apk",
                     "packageFlavor": "foss",
                 },
+                {
+                    "appName": "thunderbird",
+                    "releaseTarget": "ftp",
```

Comment:
As discussed we agree we don't need a separate job here, I think you can just do `"releaseTarget": "ftp|github"` on the github job.